client/components/editor-standalone/save-handling.ts

```diff
@@ -75,5 +77,5 @@ export const handleEmbeddedPlannings = async(
 };
 
 export const embeddedPlanningHasUnsavedChanges = () => {
-    return getEmbeddedPlanningExposed(EDITOR_TYPE.INLINE).some((x) => x.hasUnsavedChanges());
+    return getEmbeddedPlanningExposed(EDITOR_TYPE.INLINE).some((x) => x?.hasUnsavedChanges?.());
```

Comment:
would it work to return an empty array vs `[undefined, undefined]`?

Also, `.some` should return a boolean, your version sometimes returns `undefined`.